spec/search_food_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Search' do
```

Comment:
Would have loved to see VCR/webmock implemented here to stub the external api request.